
I Changed the default temp dir /tmp/javaps3media to
/tmp/javaps3media-$(USER)

===================================================================
--- a/net/pms/configuration/TempFolder.java
+++ b/net/pms/configuration/TempFolder.java
@@ -2,7 +2,6 @@
 
 import java.io.File;
 import java.io.IOException;
-
 import net.pms.PMS;
 
 import org.apache.commons.io.FileUtils;
@@ -13,8 +12,14 @@
  * @author Tim Cox (mail@tcox.org)
  */
 class TempFolder {
-
-	private static final String DEFAULT_TEMP_FOLDER_NAME = "javaps3media";
+    
+	/* 
+     Added by the Debian/Ubuntu package maintainer
+	 */
+	String username = System.getProperty("user.name");
+	//private static final String DEFAULT_TEMP_FOLDER_NAME = "javaps3media-"+username;
+	private String DEFAULT_TEMP_FOLDER_NAME = "javaps3media-"+username;
+	// End of modification modifications made by package maintainer 
 
 	private final String userSpecifiedFolder;
 	
@@ -59,7 +64,8 @@
 		return folderFile;
 	}
 
-	private static File getSystemTempFolder() throws IOException {
+		//private static File getSystemTempFolder() throws IOException {
+		private File getSystemTempFolder() throws IOException {
 		File tmp = new File(System.getProperty("java.io.tmpdir")); //$NON-NLS-1$
 		File myTMP = new File(tmp, DEFAULT_TEMP_FOLDER_NAME); //$NON-NLS-1$
 		FileUtils.forceMkdir(myTMP);
